tests/test_util.py

```diff
@@ -2,9 +2,21 @@
 
 import collections
 
+import gym
 import numpy as np
+import pytest
 
-from seals import util
+from seals import GYM_ATARI_ENV_SPECS, util
+
+
+def test_mask_score_wrapper_enforces_spec():
```

Comment:
It might be nice to add a test that actually checks `MaskScoreWrapper` masks the observations -- e.g. you could have a dummy environment that returns all-ones, a dummy mask config, and then just check that region (and only that region) is zero.

I won't insist on it though, the `MaskScoreWrapper` implementation is simple and readable already so is unlikely to have a bug, and you've already done a lot of work in this PR!